Unofficial/JavaScript/BrowserExtension/ButtonRunner/buttonRunner.js

```diff
@@ -0,0 +1,110 @@
+
```

Comment:
Could you please reformat this file to have all of the same indentation?
It's kinda all over the place lol